backend/src/models/income.ts

```diff
@@ -0,0 +1,10 @@
+export type Income = {
+  income_id: number;
+  user_id: number;
+  amount: number;
+  income_date: Date;
+  summary: string | null;
+};
+
+export type CreateIncomePayload = Omit<Income, 'user_id' | 'income_id'>;
+export type BaseIncome = Omit<Income, 'income_id'>;
```

Comment:
Base entity is the entity without the primary id field since the db will automatically populate it